File: internal/models/article_comment.go
package models

type ArticleComment struct {
	SwaggerGormModel `json:"-" gorm:"embedded"`
	Content          string `json:"content" gorm:"type:text;not null"`
	ArticleTitle     string `json:"article_title" gorm:"type:jsonb;not null"`
	IsNotify         bool   `json:"is_notify" gorm:"type:bool;not null;default:false"`
	IsRead           bool   `json:"is_read" gorm:"type:bool;not null;default:false"`
	IsPass           bool   `json:"is_pass" gorm:"type:bool;not null;default:false"`
}

File: frontend/src/interface/article-comment.ts
export interface CreateArticleComment {
    article_id: string;
    article_title: string;
    content: string;
}

import { BaseModel } from './base';

export interface ArticleComment extends BaseModel {
    content: string;
    article_title: string;
    is_notify: boolean;
    is_read: boolean;
    is_pass: boolean;
}

export interface ListCommentsRequest {
    article_id?: string;
    page?: number;
    page_size?: number;
}

export interface ListCommentsResponse {
    comments: ArticleComment[];
    total: number;
    page: number;
    page_size: number;
}